admin/server/middleware/browserify.js

```diff
@@ -1,3 +1,4 @@
+var keystone = require('../../../index');
```

Comment:
I'd like to refactor this file and add unit tests to it. now that it's being used a bit more complicatedly I think it's worth protecting.